model/app/installer.go

```diff
@@ -275,7 +274,7 @@ func (i *Installer) run() (err error) {
 	if i.man == nil {
 		panic("Manifest is nil")
 	}
-	mu := lock.ReadWrite(i.db, "app-"+i.man.Slug())
+	mu := config.GetConfig().Lock.ReadWrite(i.db, "app-"+i.man.Slug())
```

Comment:
We still have `config.GetConfig().Lock` instead of the helper `config.Lock()` here

Reply:
erf, forgotten to move all the calls indeed... Doing it now.